test/import/FSHImporter.CodeSystem.test.ts

```diff
@@ -362,48 +362,6 @@ describe('FSHImporter', () => {
         expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
       });
 
-      it('should log an error when encountering a duplicate code', () => {
```

Comment:
Ah, you're right, I hadn't added any because I saw tests for duplicate codes, but upon further inspection, they also all involve `insert` rules. I'll add some tests for handling duplicates without involving `insert` rules.